provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -0,0 +1,92 @@
+import os.path as P
```

Comment:
Not really a fan of single letter variable names.
how about s/P/osPath?

Reply:
`osPath` is way longer and doesn't differ much from the original package name `os.path`. `P` is short (shorter than the most of variable names), it starts from upper case which differentiates it from variable names)..

Reply:
Yes, but these single letter variable names are not really very intuitive. One has to go look for them what class the belong too. I don't really find them very convenient from reader's perspective. But let's not spend much time on it.